### ISSUE TITLE
omdchapter2tex.xsl

### ISSUE THREAD

**Issue by cmhughes** [openmathdocs/precalc#8]
I'm wondering about the file `omdchapter2tex.xsl`; I see that there are quite a few things in there, but I'm wondering if we can bring them into `omd2tex.xsl` and then remove `omdchapter2tex.xsl`? Or we could do it the other way round; either way, I wonder if we need both :)

**Comment by Alex-Jordan** [comment]
Yes, if you can take what I put in there and move it to omd2tex.xsl, that
would be fine. I made that at an earlier stage where I misunderstood
certain things. I thought that if I used omd2tex.xsl, that I would be
required to have a preamble appear in the tex, which I didn't want for our
chapter files.

On Sat, Dec 13, 2014 at 3:49 PM, cmhughes notifications@github.com wrote:

> I'm wondering about the file omdchapter2tex.xsl; I see that there are
> quite a few things in there, but I'm wondering if we can bring them into
> omd2tex.xsl and then remove omdchapter2tex.xsl? Or we could do it the
> other way round; either way, I wonder if we need both :)
> 
> —
> Reply to this email directly or view it on GitHub
> https://github.com/openmathdocs/precalc/issues/8.

## 

Alex Jordan
Mathematics Instructor
Portland Community College

**Comment by cmhughes** [comment]
Awesome, thanks for the update. Found a fantastic `git` command

```
git log -p xsl/omdchapter2tex.xsl
```

and it shows the history of that file, so that I can be more confident that I've caught all of the things that you've added :)

Once this is done, I wonder what you think to the idea of stripping out all of the stuff that we don't use, such as all of the preamble, `article`, `letter`, etc...?

**Comment by cmhughes** [comment]
I believe that I have moved all relevant changes from `omdchatper2tex.xsl` into `omd2tex.xsl` in the commit https://github.com/openmathdocs/precalc/commit/01890fc89975f42ed9d13f12e3ef4607d3e10a5d

To test it, I ran `xsltproc ../xsl/omd2tex.xsl functions.xml>functions.tex` and then ran `git diff chapters/functions.tex`. 

Alex, do you want to try this, and once you're happy, remove `omdchapter2tex.xsl`?

**Comment by cmhughes** [comment]
Since the new workflow (https://github.com/openmathdocs/precalc/issues/15) this issue is no longer relevant.
